src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -15,6 +15,8 @@
     <Compile Include="..\..\System.Runtime.Serialization.Xml\tests\Utils.cs" />
     <Compile Include="..\..\System.Runtime.Serialization.Xml\tests\SerializationTypes.cs" />
     <Compile Include="DataContractJsonSerializer.cs" />
+    <Compile Include="Performance\DcjsDeserializationTests .cs" />
```

Comment:
Thanks for catching this. It won't cause compilation error, but it should be removed. I'll send another PR shortly.